samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -0,0 +1,93 @@
+import { NextPage, GetStaticPaths, GetStaticProps } from 'next';
+import Error from 'next/error';
+import Layout from '../components/Layout';
+import { SitecoreContext } from '@sitecore-jss/sitecore-jss-nextjs';
+import { SitecorePageProps, extractPath } from '../lib/page-props';
+import componentFactory from '../temp/componentFactory';
+import { configBasedLayoutService as layoutService } from '../lib/layout-service';
+import { configBasedDictionaryService as dictionaryService } from '../lib/dictionary-service';
+
+const SitecorePage: NextPage<SitecorePageProps> = (props) => {
+  const { layoutData } = props;
```

Comment:
Can simplify this by moving destructuring to function params 
(ie `const SitecorePage: NextPage<SitecorePageProps> = ({ layoutData }) => {`)

Reply:
Fixed